providers/aws/aws_provider.py

```diff
@@ -322,3 +322,10 @@ def generate_regional_clients(service, audit_info):
         regional_clients[region] = regional_client
         # regional_clients.append(regional_client)
     return regional_clients
+
+
+def get_region_global_service(audit_info):
```

Comment:
Also, include the return type `-> <type>`

Reply:
Done